bin/ch-run.c

```diff
@@ -438,6 +439,9 @@ static error_t parse_opt(int key, char *arg, struct argp_state *state)
    case 'w':
       args->c.writable = true;
       break;
+   case ARGP_KEY_NO_ARGS:
+      argp_state_help(state, stderr, ARGP_HELP_STD_HELP);
```

Comment:
Looking briefly at the `argp_state_help` flags, I think we can get what we want (full help message, exit with failure), but it's not `ARGP_HELP_STD_HELP` by itself.